src/app/redux/UserSaga.js

```diff
@@ -26,7 +25,7 @@ export const userWatches = [
     uploadImageWatch,
 ]
 
-const highSecurityPages = Array(/\/market/, /\/@.+\/(transfers|permissions|password)/, /\/~witnesses/)
+const highSecurityPages = [/\/market/, /\/@.+\/(transfers|permissions|password)/, /\/~witnesses/]
```

Comment:
do not use regular expressions for testing security conditions.

Reply:
this can be handled in another PR unless you want me to do it here

Reply:
yeah, you didn't add that. i'll file an issue. disregard for now.

Reply:
discuss in https://github.com/steemit/condenser/issues/2037